### PR TITLE
Bump Cluster Autoscaler and update manifest

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

File: upup/models/cloudup/resources/addons/cluster-autoscaler.addons.k8s.io/k8s-1.15.yaml.template [hunk 1]
```diff
@@ -239,39 +273,16 @@ spec:
     metadata:
       labels:
         app: cluster-autoscaler
-      annotations:
-        prometheus.io/port: "8085"
-        prometheus.io/scrape: "true"
```
Comment (rifelpet) [inline upup/models/cloudup/resources/addons/cluster-autoscaler.addons.k8s.io/k8s-1.15.yaml.template:244]: Is there any reason not to keep these?

Comment (olemarkus) [reply upup/models/cloudup/resources/addons/cluster-autoscaler.addons.k8s.io/k8s-1.15.yaml.template:244]: Nope. I missed that in the diff. Thanks for catching

**Review by rifelpet — APPROVED** [review]

**Pull request merged**